src/services/authentication/strategies/TSPStrategy.js

```diff
@@ -0,0 +1,115 @@
+const { AuthenticationBaseStrategy } = require('@feathersjs/authentication');
+const { NotAuthenticated } = require('@feathersjs/errors');
+const { omit } = require('lodash');
+//const { jose } = require('jose');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected exception block, space or tab after '//' in comment. (spaced-comment)](https://app.codacy.com/app/schul-cloud/schulcloud-server/pullRequest?prid=4491705)